src/plugins/irc-events/whois.js

```diff
@@ -27,6 +27,9 @@ module.exports = function(irc, network) {
 				text: "No such nick: " + data.nick
 			});
 		} else {
+			// Absolute datetime in milliseconds since nick is idle
+			data.idleTime = Date.now() - data.idle * 1000;
```

Comment:
I suggest going with what we have right now and improving if/when irc-framework adds this (if possible). In the meantime, this works just fine (and also I am sooo over that PR :D).